module-2-html/exercises/01-table-of-contents/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Web design</title>
+        <body>
```

Comment:
Otvorio si `<head>` ali ga nisi zatvorio prije <body> elementa. HTML struktura zbog toga nije ispravna, morat ćeš to doraditi.